<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>JACK Audio Connection Kit - Qt GUI Interface.

   Copyright (C) 2003-2014, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

</comment>
 <class>qjackctlMessagesStatusForm</class>
 <widget class="QWidget" name="qjackctlMessagesStatusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Messages / Status - JACK Audio Connection Kit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qjackctl.qrc">:/images/messagesstatus1.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="MessagesStatusTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="MessagesTab">
      <attribute name="icon">
       <iconset resource="qjackctl.qrc">:/images/messages1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>&amp;Messages</string>
      </attribute>
      <attribute name="toolTip">
       <string>Messages log</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="MessagesTextView">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Messages output log</string>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="StatusTab">
      <attribute name="icon">
       <iconset resource="qjackctl.qrc">:/images/status1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>&amp;Status</string>
      </attribute>
      <attribute name="toolTip">
       <string>Status information</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QTreeWidget" name="StatsListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Statistics since last server startup</string>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>true</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ResetPushButton">
         <property name="toolTip">
          <string>Reset XRUN statistic values</string>
         </property>
         <property name="text">
          <string>Re&amp;set</string>
         </property>
         <property name="icon">
          <iconset resource="qjackctl.qrc">:/images/reset1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>313</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="RefreshPushButton">
         <property name="toolTip">
          <string>Refresh XRUN statistic values</string>
         </property>
         <property name="text">
          <string>&amp;Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="qjackctl.qrc">:/images/refresh1.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <tabstops>
  <tabstop>MessagesTextView</tabstop>
  <tabstop>StatsListView</tabstop>
  <tabstop>ResetPushButton</tabstop>
  <tabstop>RefreshPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qjackctl.qrc"/>
 </resources>
 <connections/>
</ui>
